server/vulnerabilities/oval/analyzer_test.go

```diff
@@ -172,6 +172,10 @@ func assertVulns(
 			continue
 		}
 
+		if !strings.HasPrefix(strings.ToLower(row[0]), "cve") {
```

Comment:
Am guessing this is the fix. Let's add a comment explaining what else is there that we are purposely skipping for now.

Reply:
Actually the fix is in [here](https://github.com/fleetdm/fleet/pull/6686/files#diff-df3f293b98148331e731892400bd2295c97fb890a2ec26177f8e5607026937baR50) and [here](https://github.com/fleetdm/fleet/pull/6686/files#diff-d188d1e7bbaeef9e2a8df73382f91c6afc39b7348e31470c9c80247f4a30ddc7R52) - by excluding non-cve entries from the scanner, existing non-cve entries will get eventually removed (because of the syncing process).

I'll add more comments to the `CveVulnerabilities` func.